esmtools/prediction.py

```diff
@@ -94,43 +82,45 @@
     lon          (y, x) float64 -47.25 -47.69 -48.12 ... 131.3 132.5 133.8
     lat          (y, x) float64 76.36 76.3 76.24 76.17 ... -77.25 -77.39 -77.54
   * ensemble     (ensemble) int64 3061 3124 3178 3023 ... 3228 3175 3144 3139
-  * year         (year) int64 1900 1901 1902 1903 1904 ... 1917 1918 1919 1920
+  * year         (year) int64 1 2 3 4 5 ... 19 20
 Dimensions without coordinates: bnds, member, vertices, x, y
 Data variables:
-    lon_bnds     (year, ensemble, member, y, x, vertices) float64 dask.array<shape=(21, 11, 9, 220, 256, 4), chunksize=(1, 1, 1, 220, 256, 4)>
-    lat_bnds     (year, ensemble, member, y, x, vertices) float64 dask.array<shape=(21, 11, 9, 220, 256, 4), chunksize=(1, 1, 1, 220, 256, 4)>
-    time_bnds    (year, ensemble, member, bnds) float64 dask.array<shape=(21, 11, 9, 2), chunksize=(1, 1, 1, 2)>
-    atmco2          (year, ensemble, member, y, x) float32 dask.array<shape=(21, 11, 9, 220, 256), chunksize=(1, 1, 1, 220, 256)>
+    tos          (year, ensemble, member, y, x) float32 dask.array<shape=(21, 11, 9, 220, 256), chunksize=(1, 1, 1, 220, 256)>
 ...
 
 This 3D example data is from curivlinear grid MPIOM (MPI Ocean Model) netcdf output.
-Time dimensions is called Years and is integer. (Original data was year
-3000-3300, now 3600. Had problems with datetime[ns] limits and xr.to_netcdf()!)
+The time dimensions is called 'year' and is in integer.
 
 """
 import os
-import esmtools as et
+from random import randint
+
 import numpy as np
 import pandas as pd
 import xarray as xr
-from pyfinance import ols
+from bs4 import BeautifulSoup
 from six.moves.urllib.request import urlopen, urlretrieve
 from xskillscore import pearson_r, rmse
-from bs4 import BeautifulSoup
-from random import randint
+
+import esmtools as et
 
 # standard setup for load dataset and examples
```

Comment:
This is fine for now, but I don't like having default variables defined in the prediction module. This will import these as global variables into every users code when they import `esmtools`.